scriptworker/constants.py

```diff
@@ -202,30 +194,26 @@
                 "path_regexes": (
                     r"^(?P<path>/comm-central)(/|$)",
                     r"^(?P<path>/releases/comm-(beta|esr\d+))(/|$)",
-                    r"^(?P<path>/(try-comm-central))(/|$)",
+                ),
+                "unstrusted_path_regexes": (
```

Comment:
sp. why do we have both path_regexes and untrusted_path_regexes?

Reply:
They aren't identical. For `comm`, there is `try-comm-central

Reply:
What benefit do we have of returning `try-comm-central`, as opposed to `None`?

Reply:
Even for unrestricted scopes, we still verify tasks, so this is needed for the dep-signing worker, and all the staging workers, to be able to generate the correct json-e context.

Reply:
Ok. Still, as noted in the first comment, `unstrusted` [sp] will make this not work. If we really want to open this up to all repos, we can probably do `r"^(?P<path>.*)/*$"` or the like.

Reply:
I ended up doing something like this. We do need to handle both
- `https://hg.mozilla.org/mozilla-central/file/<sha>/taskcluster/ci/build` -> `mozilla-central`
- `https://hg.mozilla.org/releases/mozilla-beta/file/<sha>/taskcluster/ci/build` -> `mozilla-beta`
so we do need at least a partial enumeration of things.